src/averageAges.js

```diff
@@ -15,7 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
+  let ages;
+  const mens = people.filter(person => person.sex === 'm');
+
+  if (century !== undefined) {
```

Comment:
avoid if else. make checking on century inside filter funcion and just filter people. You don't need additional map iteration